build.gradle

```diff
@@ -1,29 +1,22 @@
 plugins {
-    id 'fabric-loom' version '0.2.6-SNAPSHOT'
+    id 'fabric-loom' version '0.8-SNAPSHOT'
     id "maven-publish"
     id "signing"
 }
 
-sourceCompatibility = JavaVersion.VERSION_1_8
-targetCompatibility = JavaVersion.VERSION_1_8
+sourceCompatibility = JavaVersion.VERSION_16
+targetCompatibility = JavaVersion.VERSION_16
 
 archivesBaseName = project.archives_base_name
 version = project.mod_version
 group = project.maven_group
 
-minecraft {
-    refmapName = "refmap.reach-entity-attributes.json"
```

Comment:
The refmap name should still be explicit to remove dependency on the global project name.